src/components/searchBar/searchBar.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react'
-import './SearchBar.css'
+import './searchBar.css'
```

Comment:
I'd be curious to eventually figure out why we couldn't change the file names and have it stay capitalized when we push to github, but at least we got it working